app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.ProfileChooserViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.core.widget.NestedScrollView
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/profileChooserBackground"
+      android:overScrollMode="never"
+      android:scrollbars="none">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/profile_chooser_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
```

Comment:
Prefer avoiding the nested layouting given that we are using a ConstraintLayout (prefer aligning with constraints, instead).

Reply:
Done.